pkg/etcdcli/health_test.go

```diff
@@ -309,3 +309,106 @@ func TestGetUnhealthyMemberNames(t *testing.T) {
 		})
 	}
 }
+
+func TestIsQuorumFaultTolerant(t *testing.T) {
+	tests := []struct {
+		name         string
+		memberHealth memberHealth
+		want         bool
+	}{
+		{
+			"test all available members",
```

Comment:
(No action required) Maybe simplify with helper functions so that the test data is easier to read?

```golang
[]healthCheck{
  healthyMember("etcd-1"),
  unhealthyMember("etcd-2"),
  unstartedMember("etcd-3"),
}
```

Reply:
Yeah, tests need improved I agree. I will incorporate the comments into the final changes.